tests/Feature/CheckinTest.php

```diff
@@ -184,12 +184,13 @@ public function testCheckin() {
         // WHEN: User tries to check-in
         $response = $this->actingAs($user)
                          ->post(route('trains.checkin'), [
-                             'body'        => 'Example Body',
-                             'tripID'      => $departure->tripId,
-                             'start'       => $ibnr,
-                             'destination' => $trip['stopovers'][0]['stop']['location']['id'],
-                             'departure'   => Carbon::parse($departure->plannedWhen),
-                             'arrival'     => Carbon::parse($trip['stopovers'][0]['plannedArrival'])
+                             'body'              => 'Example Body',
+                             'tripID'            => $departure->tripId,
+                             'start'             => $ibnr,
+                             'destination'       => $trip['stopovers'][0]['stop']['location']['id'],
+                             'departure'         => Carbon::parse($departure->plannedWhen),
+                             'arrival'           => Carbon::parse($trip['stopovers'][0]['plannedArrival']),
+                             'checkinVisibility' => "0"
```

Comment:
string or integer?

Reply:
string, b/c it's a form-post-request and in forms, everything's a string.

Reply:
interesting.